src/viewer/core/ViewerPlot.C

```diff
@@ -1612,9 +1615,25 @@ ViewerPlot::GetPlotMetaData() const
             actualSpatial = *dAtts.GetThisProcsOriginalSpatialExtents();
         }
     }
+
+    /* Get bg/fg color info from window. Even though this-> object
+       appears to have it also, it is not reliably kept up to date. */
```

Comment:
This is unfortunate. Can you enter a ticket to investigate why the background and foreground colors aren't sent to the ViewerPlot reliably?